guide/sequelize/currency.md

```diff
@@ -362,6 +364,20 @@ return message.channel.send(
 	{ code: true }
 );
 ```
+</branch>
+<branch version="12.x">
+
+```js
+return message.channel.send(
+	currency.sort((a, b) => b.balance - a.balance)
+		.filter(user => client.users.cache.has(user.user_id))
+		.first(10)
+		.map((user, position) => `(${position + 1}) ${(client.users.cache.get(user.user_id).tag)}: ${user.balance}💰`)
+		.join('\n'),
+	{ code: true }
+);
+```
+</branch>
```

Comment:
Very small nit-pick, but to keep consistency with [CONTRIBUTING.md](https://github.com/discordjs/guide/blob/master/CONTRIBUTING.md) and since we had issues with non-spaced branch tags before: please keep an empty line between branch tags and any other content.